README.md

```diff
@@ -86,6 +86,7 @@ When running `cargo test`, the TypeScript bindings will be exported to the file
 | ordered-float-impl | Implement `TS` for types from *ordered_float*                                                                                                                                                             |
 | heapless-impl      | Implement `TS` for types from *heapless*                                                                                                                                                                  |
 | semver-impl        | Implement `TS` for types from *semver*                                                                                                                                                                    |
+| smol_str-impl      | Implement `TS` for types from *smol_str*                                                                                                                                                                    |
```

Comment:
Here, the feature is called `smol_str-impl`, but it's actually `smolstr-impl` in `Cargo.toml`.  

I'd prefer `smol_str-impl` everywhere, since that matches the `<crate name>-impl` naming scheme we've got going on here.

Reply:
Sure thing. I'll add it to this one #351